.github/pull_request_template.md

```diff
@@ -1,18 +1,22 @@
-## What is the purpose of the change
+## Description
```

Comment:
Should we encourage users to apply appropriate labels and assign themselves as well?

Reply:
Great idea! I'll add a comment.